lib/wasi/src/bin_factory/exec.rs

```diff
@@ -186,6 +135,130 @@ pub fn spawn_exec_module(
     Ok(join_handle)
 }
 
+fn get_start(ctx: &WasiFunctionEnv, store: &Store) -> Option<Function> {
+    ctx.data(store)
+        .inner()
+        .instance
+        .exports
+        .get_function("_start")
+        .map(|a| a.clone())
+        .ok()
+}
+
+/// Calls the module
+fn call_module(
+    ctx: WasiFunctionEnv,
+    mut store: Store,
+    handle: WasiThreadRunGuard,
+    rewind_state: Option<(RewindState, Result<(), Errno>)>,
```

Comment:
Same as other suggestion. Since the result is not used, perhaps we can pass an `Option<Errno>`

Reply:
See other comment